### ISSUE TITLE
Inject time of previous job run into lua state

### ISSUE THREAD

**Issue by ryancox** [balerter/balerter#57]
Inject the time of [previous cron execution](https://github.com/robfig/cron/blob/bc59245fe10efaed9d51b56900192527ed733435/cron.go#L63) into context of script execution. This would allow for alerts to run queries between the execution time and the time of prior execution. Perhaps create a meta module to expose these?

```lua

local meta = require("meta")
log.info("time of prior execution: " .. meta.priorExecutionTime)
```

Maybe there are some other useful bits of context to make available to scripts? Perhaps [cron location](https://github.com/robfig/cron/blob/bc59245fe10efaed9d51b56900192527ed733435/cron.go#L23)?

**Comment by negasus** [comment]
Yes. It's make sense

**Comment by negasus** [comment]
I think we need to allow the timezone for the cron to be specified in the configuration. What you think about it?

**Comment by ryancox** [comment]
Maybe but it looks like this is [configured at the top level Cron](https://github.com/robfig/cron/blob/bc59245fe10efaed9d51b56900192527ed733435/cron.go#L100-L103) and can't be changed per entry.

**Comment by negasus** [comment]
I have single Cron instance for all scripts. But maybe I can use multiple instances and configure time zone in script meta

**Comment by ryancox** [comment]
TBH I doubt it makes any difference. If you provide the last execution time and the timezone, lua script authors can do whatever math they need from there. And in the majority of cases, I bet the timezone is UTC.

**Comment by negasus** [comment]
Implemented in v0.8.3

**Comment by ryancox** [comment]
Thanks!
